qtservice/src/qtservice_unix.cpp

```diff
@@ -58,6 +58,43 @@
 #include <QMap>
 #include <QSettings>
 #include <QProcess>
+#include <QFile>
+
+static bool isServiceInited = false;
+
+static bool isSystemD()
+{
+    return QFileInfo::exists("/etc/systemd/");
+}
+
+static QString systemDPass()
```

Comment:
Is it better to name with systemDPath？

Reply:
maybe you right

Reply:
static QString systemDPass() rename static QString systemDPath()

Reply:
done